### PR TITLE
Change the text content of nic_hotplug

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1958096

Signed-off-by: Lei Yang <leiayang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Hi @PaulYuuu Could you please help review it? Thanks in advance.

**Review by PaulYuuu — APPROVED** [review]

**Comment by yanglei-rh** [comment]
Test Pass:(except rhel bug)
(03/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.q35: PASS (228.93 s)
 (04/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_repetition.q35: PASS (4792.08 s)
 (05/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_reboot.q35: PASS (310.40 s)
 (06/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_plug.q35: PASS (220.87 s)
 (07/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_unplug.q35: PASS (323.81 s)
 (08/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_system_reset.q35: PASS (315.36 s)
 (09/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_e1000e.q35: PASS (224.91 s)
 (10/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_virtio.q35: PASS (387.73 s)
 (11/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_e1000e.q35: PASS (389.06 s)
 (12/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.additional.nic_virtio.q35: PASS (121.44 s)
 (13/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.additional.nic_e1000e.q35: PASS (122.10 s)
 (18/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.vhost_nic.nic_virtio.q35: FAIL: Hotplug nic still can't get ip after reboot vm (1007.19 s)
 (19/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_virtio.q35: PASS (79.56 s)
 (20/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_e1000e.q35: PASS (79.55 s)
 (21/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nic_hotplug.with_iommu.nic_virtio.q35: FAIL: Hotplug nic still can't get ip after reboot vm (1417.58 s)
 (22/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_8139.q35: PASS (1282.89 s)
 (23/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio.q35: PASS (1240.10 s)
 (24/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_e1000e.q35: PASS (1240.00 s)
 (25/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.combination.q35: PASS (1835.41 s)
 (26/26) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.sameid.q35: PASS (664.66 s)
(03/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.q35: PASS (208.09 s)
 (04/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_repetition.q35: PASS (16010.57 s)
 (05/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_reboot.q35: ERROR: Could not verify DHCP lease: 9a:58:f3:7b:f9:9a --> 10.73.226.184 (796.32 s)
 (06/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_plug.q35: PASS (316.03 s)
 (07/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_unplug.q35: PASS (305.47 s)
 (08/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_system_reset.q35: ERROR: Could not verify DHCP lease: 9a:58:f3:7b:f9:9a --> 10.73.226.184 (810.49 s)
 (09/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_e1000e.q35: PASS (306.01 s)
 (10/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_virtio.q35: PASS (381.96 s)
 (11/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_e1000e.q35: PASS (381.12 s)
 (12/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.additional.nic_virtio.q35: PASS (106.58 s)
 (13/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.additional.nic_e1000e.q35: PASS (113.01 s)
 (18/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.vhost_nic.nic_virtio.q35: PASS (113.38 s)
 (19/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_virtio.q35: PASS (67.61 s)
 (20/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_e1000e.q35: PASS (60.71 s)
 (21/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.nic_hotplug.with_iommu.nic_virtio.q35: PASS (215.98 s)
 (22/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_8139.q35: PASS (255.77 s)
 (23/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio.q35: PASS (275.96 s)
 (24/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_e1000e.q35: PASS (277.26 s)
 (25/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.combination.q35: PASS (263.84 s)
 (26/26) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.sameid.q35: PASS (163.95 s)

**Comment by yanglei-rh** [comment]
Hi @PaulYuuu Could you help me review it again? Thanks a lot.

File: qemu/tests/nic_hotplug.py [hunk 1]
```diff
@@ -47,12 +47,11 @@ def renew_ip_address(session, mac, is_linux_guest=True):
                                                            mac)
             return None
         ifname = utils_net.get_linux_ifname(session, mac)
-        p_cfg = "/etc/sysconfig/network-scripts/ifcfg-%s" % ifname
-        cfg_con = "DEVICE=%s\nBOOTPROTO=dhcp\nONBOOT=yes" % ifname
-        make_conf = "test -f %s || echo '%s' > %s" % (p_cfg, cfg_con, p_cfg)
+        make_conf = "nmcli connection add type ethernet con-name %s ifname" \
```
Comment (PaulYuuu) [inline qemu/tests/nic_hotplug.py:50]: Although RHEL7 supports nmcli, we disabled NetworkManager in RHEL7 guest, but use network.service, so I think you should run a RHEL7 guest or use 2 command to configure RHEL7/RHEL8+

Comment (yanglei-rh) [reply qemu/tests/nic_hotplug.py:50]: I will use the current code to test rhel7 guest and update the test results.

**Comment by yanglei-rh** [comment]
Test Pass:
(3/7) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_8139: PASS (1154.57 s)
 (4/7) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio: PASS (1221.36 s)
 (5/7) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_e1000e: PASS (1178.55 s)
 (6/7) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.multi_nics_hotplug.combination: PASS (1775.06 s)
 (7/7) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.multi_nics_hotplug.sameid: PASS (665.85 s)
 (1/5) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_8139.q35: PASS (1244.92 s)
 (2/5) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio.q35: FAIL: Hotplug nic still can't get ip after reboot vm (1192.27 s)
 (3/5) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_e1000e.q35: PASS (1236.27 s)
 (4/5) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.combination.q35: PASS (1823.45 s)
 (5/5) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.sameid.q35: PASS (661.06 s)

**Comment by yanglei-rh** [comment]
@PaulYuuu  Please help review it again,I use 2 command to configure RHEL7/RHEL8+. Thanks a lot.

**Review by PaulYuuu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
```
 (01/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.arm64-pci: PASS (301.62 s)
 (02/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_repetition.arm64-pci: PASS (4603.54 s)
 (04/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_plug.arm64-pci: PASS (287.26 s)
 (05/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_virtio.with_shutdown.after_unplug.arm64-pci: PASS (382.32 s)
 (07/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_virtio.arm64-pci: PASS (541.58 s)
 (08/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.additional.nic_virtio.arm64-pci: PASS (188.26 s)
 (11/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.vhost_nic.nic_virtio.arm64-pci: PASS (191.46 s)
 (12/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_virtio.arm64-pci: PASS (157.81 s)
 (13/13) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.nic_hotplug.with_iommu.nic_virtio.arm64-pci: PASS (288.22 s)
```

**Pull request merged**